README.md

```diff
@@ -1,36 +1,11 @@
-# Toronto Mesh Logos and Stickers
+# Toronto Mesh Design
 
-## Stickers
+This repository is for all Toronto Mesh related graphic work.  Files are grouped by project.
 
-3 designs are located in `2017-stickers` based on voting for stickers to print for the [State of Our Networks 2017 event](https://tomesh.net/state-of-our-networks/).
+## File types
 
-Credits:
-- `tomesh-sticker-meshnet` [WilliamRutherford](https://github.com/WilliamRutherford)
-- `tomesh-sticker-waves` [Hamalittle](https://github.com/Hamalittle)
-- `tomesh-sticker-soon` [dcwalk](https://github.com/dcwalk)
-  - Relies on GTA boundary map by [Mortadelo2005](https://en.wikipedia.org/wiki/File:Greater_toronto_area_map.svg) CC BY-SA 3.0
+There are no explicit restrictions of graphics file types although generally open formats are favourable to proprietary ones.  Toronto Mesh website code should be directed towards the [tomesh.net](https://github.com/tomeshnet/tomesh.net) repository.
 
-<span xmlns:dct="http://purl.org/dc/terms/" property="dct:title">Toronto Mesh sticker designs</span> are licensed under a <a rel="license" href="http://creativecommons.org/licenses/by-sa/4.0/">Creative Commons Attribution-ShareAlike 4.0 International License</a>.
```

Comment:
The license block needs to stay in, and should be in the readme

Reply:
Should it stay in the readme?  I added a licenses file to keep them separate from the readme so that it didn't become real long in the future but if we want to keep them there I guess that's cool and it can just be its own section